frontend/api_postgres/entrypoint.sh

```diff
@@ -11,4 +11,6 @@ then
     echo "PostgreSQL started"
 fi
 
+docker-compose run api_postgres sh -c "python manage.py makemigrations && python manage.py migrate && python manage.py idempotent_fixtures"
```

Comment:
I think this is being run inside the container, so you shouldn't need the `docker-compose` part.

```suggestion
python manage.py makemigrations
python manage.py migrate && python manage.py idempotent_fixtures
```

I'd favor double-checking with @dwhitecl to make sure that's correct.